src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs

```diff
@@ -18,6 +18,7 @@ public partial class AttachPanel
         /// <exception cref="OutOfMemoryException">Thrown when an attempt to allocate the memory fails.</exception>
```

Comment:
please add the Obsolete tag for AttachPanel (line 9) too

Reply:
Ack